doc/decisions/0001-context-in-translations.md

```diff
@@ -18,6 +18,10 @@ For some translatable strings, we create a message context (`msgctxt`) that cont
   | `$.title` | Questionnaire title | |
   | `$.legal_basis` | Questionnaire legal basis | |
   | `$.messages` | Global answer error message | `message` rather than `messages` as each message is an individual translation |
+  | `$.hub.submission.button` | Custom hub submission button | |
+  | `$.hub.submission.guidance` | Custom hub submission guidance | |
+  | `$.hub.submission.title` | Custom hub submission title | |
+  | `$.hub.submission.warning` | Custom hub submission warning | |
```

Comment:
This file doesn't need to be updated as the most up to date list is in the Python.